.github/workflows/build-and-publish-image.yml

```diff
@@ -1,8 +1,12 @@
 name: Build Docker Image
+
 on:
   push:
+    branches: [main]
     tags:
       - v[0-9]+.[0-9]+.[0-9]+
```

Comment:
```suggestion
      - v*
```
Should we just match on everything that starts with `v`? The metadata extraction tool would probably be able to extract the correct semver data, including prerelease and build metadata

Reply:
If we want to allow pre-release stuff while still enforcing the start we could just add a wildcard to the end of the existing pattern. 
```suggestion
      - v[0-9]+.[0-9]+.[0-9]+*
```

Reply:
Yeah, you're probably right. It also makes it more strict (but not too strict) that way